src/api-v1.js

```diff
@@ -584,41 +584,39 @@ api.declare({
   name: 'createAmiSet',
   deferAuth: true,
   scopes: [['aws-provisioner:manage-ami-set:<amiSetId>']],
-  input: 'create-ami-set-request.json#',
-  output: 'get-ami-set-response.json#',
   title: 'Create new AMI Set',
   stability:  base.API.stability.stable,
   description: [
     'Create an AMI Set. An AMI Set is a collection of AMIs with a single name.',
   ].join('\n'),
 }, async function (req, res) {
   let input = req.body;
-  let amiSet = req.params.amiSetId;
+  let AmiSetId = req.params.id;
 
   input.lastModified = new Date();
 
   // Authenticate request with parameterized scope
-  if (!req.satisfies({amiSet: amiSetId})) {
+  if (!req.satisfies({amiSet: AmiSetId})) {
     return;
   }
 
   // Create amiSet
-  let aSet;
+  let amiSetId;
```

Comment:
this *will* refer to an AmiSet instance, so let's call it amiSet.

Basically, if a thing refers to a class (or in JS terms a constructor), it should be Caps

Here are some examples:
```javascript
class ExampleClass { }

function ExampleConstructor () { }

let ExampleClass = require('./lib/example-class').ExampleClass;
let exampleInstance = new ExampleClass();

let a = 'john';
```